app/controllers/manage/SetController.php

```diff
@@ -109,21 +109,33 @@ public function ListSets() {
 			switch($pn_mode) {
 				case 0:
 				default:
-					$va_set_list['access'] = __CA_SET_EDIT_ACCESS__;
+					$set_params['access'] = __CA_SET_EDIT_ACCESS__;
 					break;
 				case 1:
-					$va_set_list['allUsers'] = true;
+					$set_params['allUsers'] = true;
 					break;
 				case 2:
-					$va_set_list['publicUsers'] = true;
+					$set_params['publicUsers'] = true;
 					break;
 			}
 		} else {
-			$va_set_list['access'] = __CA_SET_EDIT_ACCESS__;
+			$set_params['access'] = __CA_SET_EDIT_ACCESS__;
 		}
 
+		$sql_sorts = array("cs.set_id", "csl.name", "clil.name_singular", "cs.access", "u.lname", "cs.access", "cs.status", "cs.rank");
```

Comment:
Remove duplicate element.
```suggestion
		$sql_sorts = array("cs.set_id", "csl.name", "clil.name_singular", "cs.access", "u.lname", "cs.status", "cs.rank");
```

Reply:
Well :octocat: if you want to treat the text change like that I guess you can :confounded: